tripy/tripy/constants.py

```diff
@@ -16,6 +16,12 @@
 #
 
 CONSTANT_IR_PRINT_VOLUME_THRESHOLD = 5
+
+
+# Aim is to exclude shape tensors, and TensorRT shape dims supports up to 8 dimensions
+# so threshold value should be a lot bigger than 8
+STROGE_OP_CACHE_VOLUME_THRESHOLD = 64
```

Comment:
```suggestion
STORAGE_OP_CACHE_VOLUME_THRESHOLD = 64
```

Reply:
We should also move this up or down - currently it's splitting `CONSTANT_IR_PRINT_VOLUME_THRESHOLD` from its docstring

Reply:
moved